todoApp/spec/main.tsp

```diff
@@ -238,6 +238,20 @@ namespace TodoItems {
     @statusCode statusCode: 422;
   }
 
+  /**
+   * We need to decorate NotFoundResponse with an explicit `@error` decorator so it's clear that
+   * it is not a valid response
+   */
+  @error
+  model NotFoundErrorResponse is NotFoundResponse;
+
+  /**
+   * We need to decorate NoContentResponse with an explicit `@error` decorator so it's clear that
+   * it is not a valid response
+   */
+  @error
+  model NoContentErrorResponse is NoContentResponse;
```

Comment:
NoContentResponse is 204, which seems not an error?

Reply:
Agreed, this should be a success response.

Reply:
good point, I'm not sure why I did that haha, fixing now, thanks!